_posts/en/newsletters/2024-05-15-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #302'
+permalink: /en/newsletters/2024/05/15/
+name: 2024-05-15-newsletter
+slug: 2024-05-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Release of utreexod beta:** Calvin Kim [posted][kim utreexo] to the
+  Bitcoin-Dev mailing list to announce the beta release of `utreexod`, a
+  full node with support for [utreexo][topic utreexo].  Utreexo allows a
+  node to store a small commitment to the state of the UTXO set rather
+  than the entire set itself; for example, a minimal commitment can be
+  32 bytes and the current full set is about 12 GB, making the
+  commitment on the order of about a billion times smaller.  To reduce
+  bandwidth, utreexo may store additional commitments, increasing its
+  use of disk space, but still keeping its chainstate roughly on the
+  order of a million times smaller than a traditional full node.
```

Comment:
Is this sentence on storing additional commitments an interpretation of this bullet point from the ML post?

> - Efficient P2P transaction relay with support for caching utreexo proofs for
  mempool transactions.

If so, then it seems wrong to say "utreexo may store additional commitments", since utreexo by itself is a deterministic accumulator that stores the minimum amount of data necessary. It would be more accurate to say "**a utreexo node** may **cache** additional commitments". The ML post also doesn't say whether these are cached on disk or in memory, so I would double check the claim that it uses more disk space.

Reply:
No, it's not derived from that line in the ML post.  It's a simple observation that utreexo can save an extraordinary amount of bandwidth by storing a few kilobytes to megabytes of the intermediate merkle nodes near the root of the tree, instead of including them with every transaction.

I don't know if Kim's implementation does that, but we're talking about utreexo in general here.